js/Store.js

```diff
@@ -31,6 +31,20 @@ function reducer(state, action) {
     }
     return newState;
 
+  case 'setView':
+    newState.currentView = action.view;
+    return newState;
+
+  case 'seeking':
+    newState.seeking = true;
```

Comment:
oh. shouldn't that be "false"? also do you want to set it back to "false" when the game starts?